modules/react/form-field/lib/Label.tsx

```diff
@@ -72,8 +72,6 @@ const RequiredAstrisk = styled('abbr')({
 });
 
 // Used inside the fieldset component instead of a label for accessible radio groups.
-// Extra `width: 100%` style is to help IE11 wrap text properly in a top-positioned
-// FormField's legend element.
 const LegendComponent = styled('legend')<LabelProps>(...labelStyles, ({labelPosition}) => {
   if (labelPosition === Label.Position.Top) {
```

Comment:
Can this go away?

Reply:
When this was removed, chromatic caught a regression in chrome, so I left it in